client/lib/checkout/validation.js

```diff
@@ -204,17 +205,20 @@ validators.validExpirationDate = {
 	error: validationError,
 };
 
-validators.validCPF = {
+validators.validBrazilTaxId = {
 	isValid( value ) {
 		if ( ! value ) {
 			return false;
 		}
-		return isValidCPF( value );
+		return isValidCPF( value ) || isValidCNPJ( value );
 	},
 	error: function( description ) {
-		return i18n.translate( '%(description)s is invalid. Must be in format: 111.444.777-XX', {
-			args: { description: description },
-		} );
+		return i18n.translate(
+			'%(description)s is invalid. Must be in format: 111.444.777-XX or 11.444.777/0001-XX',
```

Comment:
:disappointed: 7 existing translations will be lost with this change.

Reply:
That's ok :)

Reply:
What about we make this a variable `%(description)s is invalid. Must be in format: %(validFormat)`?